### ISSUE TITLE
The harbor deployed by kk requires a custom domain name + certificate and later updated domain name

### ISSUE THREAD

**Issue by shenhonglei** [kubesphere/kubekey#1411]
### Your current KubeKey version

2.*

### Describe this feature

Change the domain name of the harbor deployed by kk?
Or can have customized planning, some customers need to use enterprise domain name suffix

### Describe the solution you'd like

1.we can define the harbor domain name by yourself through kk
2.The deployed harbor domain name can be modified through kk

**Comment by xiaods** [comment]
welcome pr

**Comment by xiaods** [comment]
I don't think this is good proposal, the harbor is internal image registry, don't need share with enterprise registry requirement. so @shenhonglei please consider add any another reason ?

**Comment by liubarnabas** [comment]
This is an important and necessary requirement for service mesh management of multiple-clusters .
App1 in the Cluster1 need expose its service svc1 ( svc1.cluster1.local) to App2 in the cluster2 for accessing.
 In this case, k8s Cluster1 and Cluster2 must configure different domain name for service mesh.

Hope this requirement could be solve as soon as possible.
